src/i18n/locales/en-US.json

```diff
@@ -76,7 +76,9 @@
       "success-tooltip-aria": "Saved successfully",
       "saveArea-tooltip": "Save Area",
       "confirm-success": "Okay",
-      "cancel-save": "Don't save"
+      "cancel-save": "Don't save",
+      "invalid-headline-text": "Invalid Edits",
+      "invalid-text": "The segments do not create a valid video. Either change or discard your edits if you wish to proceed. If you wanted to delete this video use the Opencast Admin UI. Contact an adminstrator for further help."
```

Comment:
We should avoid mentioning the Admin UI. The editor can be accessed from other frontends as well, like Tobira or LMS plugins. User may not know what the "Opencast Admin UI" is, nor should they need to.